pmm-app/src/pmm-check/components/FailedChecksTab/FailedChecksTab.utils.ts

```diff
@@ -0,0 +1,26 @@
+import { logger } from '@percona/platform-core';
+import { SHOW_SILENCED_VALUE_KEY, SHOW_SILENCED_DEFAULT } from './FailedChecksTab.constants';
+
+export const loadShowSilencedValue = (): boolean => {
+  try {
+    const showSilencedValue = window.localStorage.getItem(SHOW_SILENCED_VALUE_KEY);
+
+    if (showSilencedValue === null) {
+      return SHOW_SILENCED_DEFAULT;
+    }
+
+    return showSilencedValue === 'true';
```

Comment:
I would do here: `return !!showSilencedValue` and remove the condition. If `showSilencedValue` is null, undefined or false this will return false (which is the default), if it's 'true' it will return true. That would simplify this code. Also consider moving the return out of the catch.

Reply:
But if you store 'false', then it'd be true :)

Reply:
Indeed, thanks for pointing it, it's better this way then!